DESCRIPTION

```diff
@@ -48,6 +48,7 @@ Remotes:
 RoxygenNote: 6.1.1
 Suggests:
     knitr,
+    pkgdown,
```

Comment:
idk if this is required.  All times we use `pkgdown` are before we submit to cran.  

I'm inclined to say we do not use this line.

Reply:
I put it there because I thought it's handy that `devtools::install_dev_deps()` can install everything necessary to work on the package, including producing the site.

If we shouldn't `Suggest` packages that are not used after cran, should we then also remove `testthat`? That seems funky to me.

Reply:
`testthat` is used by travis and by cran.

I'd be happy to upgrade our docs script to check for the package and install it if it is not there.

Reply:
Thanks, that sounds good. I personally just find it unintuitive for our list of necessary dependencies to be distributed among multiple files, but I also don't feel too strongly about it.